### PR TITLE
Update navigation and avatars

### PR DESCRIPTION
**Pull request by kevinch**
#### What is the purpose of this pull request?
Fix navigation & remove avatars.

#### What problem is this solving?
Broken nav & avatars.

#### How to test it?
https://example--teamadmin.myvtex.com/admin/example

#### Screenshots or example usage
n/a

#### Types of changes

- [ ] Chore (non-breaking change which doesn't change any functionalities)
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: admin/routes.json [hunk 1]
```diff
@@ -5,6 +5,6 @@
   },
   "admin.app.example-detail": {
     "component": "AdminExampleDetail",
-    "path": "/admin/app/example/:id"
```
Comment (marcelovicentegc) [inline admin/routes.json]: I find this is useful to let people know straight away they can do this

File: .github/workflows/danger.yml [hunk 2]
```diff
@@ -1,22 +0,0 @@
...
-          node-version: 12.x
-        env:
-          RUNNER_TEMP: /tmp
-      - name: yarn install
-        run: yarn install
-      - name: Danger CI
-        uses: vtex/danger@master
-        env:
-          GITHUB_TOKEN: '${{ secrets.GITHUB_TOKEN }}'
-          REQUIRE_CHANGELOG_VERSION: false
```
Comment (marcelovicentegc) [inline .github/workflows/danger.yml:22]: These workflows seem useful, why removing them?

File: admin/routes.json (hunk 1 above)
Comment (kevinch) [reply admin/routes.json]: I plan to add a working example soon, so i removed it for now in order to get it all working and non broken.

File: .github/workflows/danger.yml (hunk 2 above)
Comment (kevinch) [reply .github/workflows/danger.yml:22]: DangerJS has never been set as a pattern in VTEX and the tests made with it brought more questions and problems than solutions.

File: admin/routes.json (hunk 1 above)
Comment (marcelovicentegc) [reply admin/routes.json]: Got it! Doit!

File: .github/workflows/danger.yml (hunk 2 above)
Comment (marcelovicentegc) [reply .github/workflows/danger.yml:22]: I see your point! What about the build/test workflows? It occurs to me that we could replace them with a Drone example, but that's out of the scope of this PR

Comment (kevinch) [reply .github/workflows/danger.yml:22]: Interesting. I'd say it belongs more to the internal example we're building.

File: package.json [hunk 3]
```diff
@@ -29,6 +29,7 @@
     "lint-staged": "^10.2.2",
     "prettier": "^2.0.5",
     "typescript": "^3.8.3",
-    "@vtex/danger": "^0.2.7"
+    "@vtex/danger": "^0.2.7",
```
Comment (marcelovicentegc) [inline package.json]: We can remove this dependency as well, right?

**Review by marcelovicentegc — APPROVED** [review]
LGTM 🚀

File: .github/workflows/danger.yml (hunk 2 above)
Comment (marcelovicentegc) [reply .github/workflows/danger.yml:22]: Good point!

File: package.json (hunk 3 above)
Comment (kevinch) [reply package.json]: Good catch.

Comment (kevinch) [reply package.json]: Updated

**Pull request merged**